File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// FoodHygieneRating SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class FoodHygieneRatingMakeContext
{
    public static function call(array $ctxmap, ?FoodHygieneRatingContext $basectx): FoodHygieneRatingContext
    {
        return new FoodHygieneRatingContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# FoodHygieneRating SDK utility: make_context
require_relative '../core/context'
module FoodHygieneRatingUtilities
  MakeContext = ->(ctxmap, basectx) {
    FoodHygieneRatingContext.new(ctxmap, basectx)
  }
end
